src/main/java/az/edu/turing/module02/part01/lesson01/PlayerApp.java

```diff
@@ -0,0 +1,51 @@
+package az.edu.turing.module02.part01.lesson01;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class PlayerApp {
+    public static void main(String[] args) {
+        List<Player> players = getPlayers();
+        printList(players);
+
+        Collections.sort(players, (Player o1, Player o2) -> {
+            if (o1.getScore() != o2.getScore()) return Double.compare(o2.getScore(), o1.getScore());
```

Comment:
we can rewrite this with method reference using Comparator chain methods, pls investigate